comments/migrations/0006_auto_20200331_1052.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-31 14:52
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('comments', '0005_comment_deleted'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='comment',
+            name='level',
+            field=models.PositiveIntegerField(editable=False),
+        ),
+        migrations.AlterField(
+            model_name='comment',
+            name='lft',
+            field=models.PositiveIntegerField(editable=False),
+        ),
+        migrations.AlterField(
+            model_name='comment',
+            name='rght',
+            field=models.PositiveIntegerField(editable=False),
+        ),
+    ]
```

Comment:
I'm assuming this migration is here because our comments app is installing the latest django-mptt.  I think we can keep this migration, but we should pin to a specific version of django-mptt

Reply:
Updated